src/components/ClusterNodes/NodeDetails.tsx

```diff
@@ -0,0 +1,765 @@
+                            Annotation ({nodeDetail.annotations.length})
+                        </div>
+                        {selectedSubTabIndex == 1 && <div className="node-details__active-tab" />}
+                    </li>
+                    <li
+                        className="tab-list__tab pointer"
+                        onClick={() => {
+                            setSelectedSubTabIndex(2)
+                        }}
+                    >
+                        <div className={`mb-6 fs-13${selectedSubTabIndex == 2 ? ' fw-6 cb-5' : ' fw-4'}`}>
+                            Taints ({nodeDetail.taints?.length || 0})
+                        </div>
+                        {selectedSubTabIndex == 2 && <div className="node-details__active-tab" />}
+                    </li>
+                </ul>
+                <div className=" pr-20 pl-20 pt-12 pb-12">
+                    {selectedSubTabIndex == 0 && renderLabelTab()}
+                    {selectedSubTabIndex == 1 && renderAnnotationTab()}
+                    {selectedSubTabIndex == 2 && renderTaintTab()}
+                </div>
+            </div>
+        )
+    }
+    const renderErrorOverviewCard = (): JSX.Element => {
+        const nodeErrorKeys = Object.keys(nodeDetail.errors)
+        if (!nodeErrorKeys.length) return null
+        return (
+            <div className="mb-12 en-2 bw-1 br-4 bcn-0">
+                <div className="flexbox bcr-5 pt-12 pb-12 pr-10 pl-20 top-radius-4">
+                    <Error className="error-icon-white mt-2 mb-2 mr-8 icon-dim-18" />
+                    <span className="fw-6 fs-14 cn-0">
+                        {nodeErrorKeys.length === 1 ? '1 Error' : nodeErrorKeys.length + ' Errors'}
+                    </span>
+                </div>
+                <div className="pt-12 pr-20 pl-20">
+                    {nodeErrorKeys.map((key) => (
+                        <>
+                            <div className="fw-6 fs-13 cn-9">{key}</div>
+                            <p className="fw-4 fs-13 cn-7 mb-12">{nodeDetail.errors[key]}</p>
+                        </>
+                    ))}
+                </div>
+            </div>
+        )
+    }
+    const renderProbableIssuesOverviewCard = (): JSX.Element => {
+        const isCPUOverCommitted = Number(cpuData.usagePercentage.slice(0, -1)) > 100
+        const issueCount =
+            (isCPUOverCommitted ? 1 : 0) + (nodeDetail.unschedulable ? 1 : 0) + (nodeDetail.taintCount > 0 ? 1 : 0)
+        if (!issueCount) return null
+        return (
+            <div className="mb-12 en-2 bw-1 br-4 bcn-0">
+                <div className="flexbox bcy-5 pt-12 pb-12 pr-10 pl-20 top-radius-4">
+                    <AlertTriangle className="alert-icon-white mt-2 mb-2 mr-8 icon-dim-18" />
+                    <span className="fw-6 fs-14 cn-9">
+                        {issueCount === 1 ? '1 Probable issue' : issueCount + ' Probable issues'}
```

Comment:
Suggestion - can also be written as `${issueCount} Probable issue${issueCount > 1 : 's' : ''}`.

Reply:
Done